addons/http_routing/views/http_routing_template.xml

```diff
@@ -13,14 +13,14 @@
         </t>
     </template>
 
-    <template id="error_message">
+    <template id="error_message" name="Error Message">
```

Comment:
Not really useful on this one, it's never called as main template but only `t-call`ed from other template.
But I guess it doesn't hurt to have a name on it

Reply:
hi @rdeodoo ,
thanks and reverted